libsolidity/lsp/LanguageServer.cpp

```diff
@@ -181,6 +206,18 @@ void LanguageServer::compile()
 	FileRepository oldRepository(m_fileRepository.basePath(), m_fileRepository.includePaths());
 	swap(oldRepository, m_fileRepository);
 
+	// Load all solidity files from project.
+	if (m_analyzeAllFilesFromProject)
+		for (auto const& projectFile: allSolidityFilesFromProject())
+		{
+			lspDebug(fmt::format("adding project file: {}", projectFile.generic_string()));
+			m_fileRepository.setSourceByUri(
+				m_fileRepository.sourceUnitNameToUri(projectFile.generic_string()),
+				util::readFileAsString(projectFile)
+			);
+		}
+
+	// Overwrite all files as opened by the client and might potentially have changes.
```

Comment:
Seems like something is missing in this sentence:
```suggestion
	// Overwrite all files as opened by the client, including the ones which might potentially have changes.
```